tests/alfacase/test_plugin_alfacase_to_case.py

```diff
@@ -1073,3 +1064,87 @@ def test_load_not_installed_plugin(datadir):
 
     case = convert_alfacase_to_description(alfacase)
     assert case.plugins == []
+
+
+def test_load_plugin_without_version(
+    datadir: Path, abx_plugin: None, monkeypatch: MonkeyPatch
+) -> None:
+    """
+    Test to ensure the backward compatibility in case of version is not specifeid in the
+    .alfacase file.
+    """
+    plugins_root = datadir / "test_plugins"
+    monkeypatch.setenv("ALFASIM_PLUGINS_DIR", str(plugins_root))
+
+    assets = plugins_root / "abx-1.0.0" / "assets"
+    assets.mkdir()
+
+    plugin_yaml = assets / "plugin.yaml"
+    plugin_yaml.touch()
+
+    alfacase = datadir / "test.alfacase"
+    alfacase.touch()
+    alfacase.write_text(
```

Comment:
```suggestion
    alfacase.write_text(
```

I think we don't need to create the file before writing to it.